consensus/consensus_types/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "consensus-types"
+version = "0.1.0"
+authors = ["Libra Association <opensource@libra.org>"]
+license = "Apache-2.0"
+publish = false
+edition = "2018"
+
+[dependencies]
+bytes = "0.4.12"
+mirai-annotations = { version = "1.4.0", default-features = false }
+proptest = { version = "0.9.4", optional = true }
+protobuf = "~2.7"
```

Comment:
You might need to do a bit more cleanup given we no longer use the protobuf crate anymore and instead use prost